### ISSUE TITLE
[REQUEST]: Remove the dependency on node for the signaling part

### ISSUE THREAD

**Issue by qleguennec** [Unity-Technologies/UnityRenderStreaming#735]
### Is your feature request related to a problem?

We need to ship a version of this codebase in an executable. The node dependency increases the build size and complicates the build process.

### Describe the solution you'd like

The signaling part of this solution could be done in C# in Unity. No node dependency would be required.

**Comment by karasusan** [comment]
@qleguennec 
You mean that remove the node dependeny from the signaling server?

**Comment by qleguennec** [comment]
@karasusan Yes. Ideally, this would have the effect that we wouldn't need to launch the signaling server prior to using webrtc. The signaling server will be embedded in Unity and would start alongside the rest of the startup scripts.

**Comment by karasusan** [comment]
@qleguennec
In other words your idea, Unity runtime should work as a http server for signaling, is it right? This idea is not bad, but I am not sure that how many users hope to realize that.

**Comment by qleguennec** [comment]
@karasusan Just to clarify your opinion, it is based on people that will be able to contribute to the repo? Or on the principle of least astonishment? I'm not sure I understand the base reasoning around your comment.

**Comment by karasusan** [comment]
@qleguennec
This is no basis. 

I understood your opnition. 
This is that Unity runtime integrate the signaling server, and reduce the footprint of node server. Surely, it would be valuable as a one option of the signaling implementation.

**Comment by qleguennec** [comment]
@karasusan Okay, I understand.

**Comment by karasusan** [comment]
memo: URS-501
